evap/static/scss/components/_buttons.scss

```diff
@@ -152,6 +152,16 @@ a:not([href]):not(.disabled) {
     min-width: 140px;
 }
 
+.button-notebook {
+    @include media-breakpoint-up(md) {
```

Comment:
Currently, when in portrait mode, the button is at the end of the site, which is a bit odd to find. Should we put it at the top? @janno42

Reply:
if we would fix this somewhere it would overlap with evap content.

Reply:
It's probably better to place the icon on the top of the page (above the headline) and not attached to the edge but as a normal button.

Reply:
now the position changes with the notebook break